client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -197,12 +190,12 @@ describe( 'BackupDatePicker', () => {
 				done();
 			};
 
-			const picker = shallow(
-				<BackupDatePicker selectedDate={ selectedDate } onDateChange={ onDateChange } />
-			);
+			render( <BackupDatePicker selectedDate={ selectedDate } onDateChange={ onDateChange } /> );
 
-			const button = picker.find( '.backup-date-picker__select-date--previous' );
-			button.simulate( 'keydown', { key: ' ' } );
+			const button = screen.getByText( 'Dec 31, 2019' );
+			button.focus();
+
+			fireEvent.keyDown( button, { code: 'Space', key: ' ' } );
```

Comment:
Why are we not using `user.keyboard()` from `@testing-library/user-event` here? 

This looks like an opportunity to fix the underlying issue, and start relying on the universal `event.code` instead. See https://github.com/WordPress/gutenberg/pull/43437/ for a similar fix.

This applies to the rest of the PR where we use `fireEvent`.

Reply:
Hi @tyxla, I have updated the component to use `code` and also updated the test so that we can `await` before the assertion, yet the keyboard event is not getting triggered, I have pushed the changes with what I tried. I hope you can take a look and help me figure out what I am missing. Thanks

Reply:
Thanks, I'm going to leave a separate comment in my review.